sopel-modules/rat-board.py

```diff
@@ -910,6 +910,40 @@ def cmd_list(bot, trigger, params=''):
         -@: Show full case IDs.  (LONG)
 
     """
+    count = 0
+    plats = []
+    params = ['']
+    tmp = ''
+    for x in remainder:
+        if x in ['@', 'i', 'r', 'u']:
+            params[0] = '-'
+            params.append(x)
+        else:
+            plats.append(x)
+
+    tmpStr = ''
+    for x in plats:
+        tmpStr += x
+
+    offset = 0
+    tmp = list(tmpStr)
+    for x in range(0, len(tmpStr)):
+        if (x + offset) % 3 != 0 or x == 0: continue
+        if tmp[x + offset] != ' ':
+            tmp.insert(x + offset - 1, ' ')
+            offset += 1
+    tmpStr = ''.join(tmp)
+    plats = tmpStr.split(' ')
+    showpc = 'pc' in plats
+    showps = 'ps' in plats
+    showxb = 'xb' in plats
+    showAllPlats = True if not (showpc or showps or showxb) else False
+
+    showPlats = []
+    showPlats.append("pc") if showpc else None
```

Comment:
should be replaced with
```python
showpc and showPlats.append("pc")
```
The `Else none` is introducing needless complexity.
(the append only occurs if showpc is truthy)

Reply:
done